extensions/Worker.Extensions.CosmosDB/src/Config/CosmosDBBindingOptionsSetup.cs

```diff
@@ -14,11 +14,13 @@ internal class CosmosDBBindingOptionsSetup : IConfigureNamedOptions<CosmosDBBind
     {
         private readonly IConfiguration _configuration;
         private readonly AzureComponentFactory _componentFactory;
+        private readonly IOptions<WorkerOptions> _workerOptions;
 
-        public CosmosDBBindingOptionsSetup(IConfiguration configuration, AzureComponentFactory componentFactory)
+        public CosmosDBBindingOptionsSetup(IConfiguration configuration, AzureComponentFactory componentFactory, IOptions<WorkerOptions> workerOptions, )
```

Comment:
This might not really have any impact, but just for accuracy:

Since this is being registered as `singleton`, it would be better to import `IOptionsMonitor<WorkerOptions>` and access it via `.CurrentValue`. I say this has no real impact because when you get down to it, we only access the `ObjectSerializer` and that is unlikely to change over the lifetime of this process. But it is good habit to ensure that options reacts to changes of config and other options.